src/main/java/org/takes/http/Options.java

```diff
@@ -199,4 +186,38 @@ public long maxLatency() {
         }
         return msec;
     }
+
+    /**
+     * Convert the provided arguments into a Map. The expected
```

Comment:
@essobedo this is really verbose. You've repeated implementation almost fully. Code should be the only source of true :)